lib/child-writer/index.js

```diff
@@ -170,14 +182,102 @@ exports.write = (image, drive, options) => {
     }
   }
 
+  // Timeout function that only runs if we exceed the specified timeout,
+  // terminating the server.
+  // @type {Timeout}
+  let terminationTimeout = null
+
+  // @type {Object} state
+  // @type {ChildProcess | undefined} [state.child]
+  // @type {Socket | undefined} [state.childSocket]
+  // @type {Boolean | undefined} [state.isChildRunning]
+  // @type {Boolean | undefined} [state.isChildSocketConnected]
```

Comment:
Shouldn't this and the above be in the fashion of

```js
/**
 * [state description]
 * @private
 * @type {Object} state
 * @type {ChildProcess | undefined} [state.child]
 * @type {Socket | undefined} [state.childSocket]
 * @type {Boolean | undefined} [state.isChildRunning]
 * @type {Boolean | undefined} [state.isChildSocketConnected]
 */
```

Reply:
Yup, I forgot this as I copy-pasted the lines.